src/runtime/ExtendedNativeFunctionObject.cpp

```diff
@@ -0,0 +1,11 @@
+#include "Escargot.h"
+#include "ExtendedNativeFunctionObject.h"
+
+namespace Escargot {
+
+ExtendedNativeFunctionObject::ExtendedNativeFunctionObject(ExecutionState& state, NativeFunctionInfo info, SmallValueVector values)
+    : NativeFunctionObject(state, info)
+{
+    m_values = values;
```

Comment:
IMO it would be better to initialize `m_value` by calling the [move constructor](https://github.com/Samsung/escargot/blob/81c8c005cff5b645443b7b11d1cc827007a484ba/src/util/Vector.h#L123) of Vector. Move constructor could reduce the overhead of Vector copying.